CMakeLists.txt

```diff
@@ -1,75 +1,85 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(moveit2_tutorials)
 
-set(CMAKE_CXX_STANDARD 14)
+if(NOT "${CMAKE_CXX_STANDARD}")
```

Comment:
This could use the `moveit_package` macro https://github.com/ros-planning/moveit2/blob/main/moveit_common/cmake/moveit_package.cmake

Reply:
That is where I grabbed the current if statement from. I will get that added in.